sources/assets/zsh/aliases.d/ysoserial

```diff
@@ -1 +1 @@
-alias ysoserial='JAVA_HOME=/usr/lib/jvm/java-11-openjdk java -jar /opt/tools/ysoserial/ysoserial.jar'
+alias ysoserial='JAVA_HOME=/usr/lib/jvm/java-17-openjdk java -jar /opt/tools/ysoserial/ysoserial.jar'
```

Comment:
Not sure about this, as per the following issue https://github.com/frohoff/ysoserial/issues/203, we probably want to this comment: https://github.com/frohoff/ysoserial/issues/203#issuecomment-1691067536
Also, we have https://github.com/ThePorgs/Exegol-images/issues/234 open
Can you modify ysoserial's test command with something like `ysoserial CommonsCollections4 'whoami'` so that we make sure everything works as intended?